mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -177,35 +182,37 @@ def dialog_renderer(self):
         # Fall back to main language
         return self.dialog_renderers.get(self._core_lang)
 
+    @property
+    def settings_path(self):
+        return join(xdg.BaseDirectory.save_config_path(BASE_FOLDER, 'skills', self.skill_id), 'settings.json')
+
+    @property
+    def settings_write_path(self):
+        # hopefully nothing in the outside world ever noticed this was not underscore prefixed
+        LOG.warning("self.settings_write_path has been deprecated! do not use it, it is an internal variable")
```

Comment:
Consider consolidating skill FS paths as discussed https://github.com/MycroftAI/mycroft-core/issues/2912

Reply:
Also, this is a breaking change in NeonCore
 https://github.com/NeonGeckoCom/neon-skill-utils/blob/dc93b9be163af5339361e03d85bd79bf67857e45/neon_utils/skills/mycroft_skill.py#L75

Reply:
that would go against the XDG standard we have been pushing towards.... imho skill settings should only live in the XDG path, even in neon-core

still can't make setting_write_path a property, will keep it a variable and add deprecation warnings at least